Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:14
+
+EXPOSE 8079
+
+WORKDIR /usr/src/app
+
+COPY package.json .
+RUN yarn install
+
+COPY . .
+RUN yarn build
+

```

Add another `FROM` instruction to create the next build stage.
```suggestion
FROM node:14
ENV NODE_ENV=production
WORKDIR /usr/src/app-prod
COPY /usr/src/app/package.json .
RUN yarn install
COPY /usr/src/app/dist ./dist/
```